src/views/LoginForm/LoginForm.jsx

```diff
@@ -0,0 +1,171 @@
+              <FormInput
+                className="mb-2"
+                type={inputs.type}
+                name={inputs.name}
+                id={inputs.name}
+                placeholder={inputs.placeholder}
+              />
+            </FormGroup>
+          ))}
+        </div>
+        <FormGroup>
+          <Button
+            size="lg"
+            pill
+            theme="dark"
+            onClick={event => handleButtonClick(event)}
+            name="submit"
+          >
+            {"Login"}
+          </Button>
+        </FormGroup>
+      </Form>
+
+     { ["email", "password"].map((field, key) => (
+       document.getElementById(field) && document.getElementById(field)!== ""  && //If not test fails No DOM elements were found for #email.
+        <Tooltip
+          key={key}
+          open={hasErrors[field].hasError}
+          target={`#${field}`}
+        >
+          <span id="input_error"> {hasErrors[field].message} </span>
+        </Tooltip>
+      ))}
+    </div>
+    
+  );
+};
+
+export const defaultErrors = {
+  email: {
+    hasError: false,
+    message: ""
+  },
+  password: { hasError: false, message: "" }
+};
+
+export default props => {
+
+  const [user_credentials, setUserCredential] = useState({});
+  const [error_message, setErrorMessage] = useState("");
+  const [hasErrors, setHasErrors] = useState(defaultErrors);
+  const [alertMessage, setAlertMessageFields] = useState({
+    visible: false,
+    alertType: "danger"
+  });
+
+  /**
+   * Triggers the error alert banner.
+   * !!!! Temporary put a success alert for successfull login.
+   * @param {String} alertType the alert type success or error.
+   */
+  const triggerAlert = alertType => {
+    setAlertMessageFields({
+      visible: !alertMessage.visible, //
+      alertType: [alertType]
+    });
+  };
+  /**
+   * @function handleResponse
+   * @description filters the error type and setting the erro message and form tip error.
+   * @param {Object} response the Http response from backend.
+   */
+  const handleResponse = response => {
+    if (response.status === "BAD_REQUEST") {
+      const errors = response.errors;
+      errors.forEach(error =>
+        setHasErrors({
+          ...hasErrors,
+          [error.field]: {
+            ...hasErrors[error.field],
+            hasError: true,
+            message: error.message
+          }
+        })
+      );
+    };
+
+    if (response.status === "BAD_REQUEST" || response.status === "UNAUTHORIZED")
+      setAlertMessageFields({ visible: true, alertType: "danger" });
+
+    return setErrorMessage(response.message);
+  };
+  /**
+   * @function handleButtonClick
+   * @description Handles the submission, calls the backend API, through login function.
+   * handle the response with handleResponse function.
+   * @param {Event} event
+   */
+  const handleButtonClick = async event => {
+    event.preventDefault();
+    await login(user_credentials).then(response => {
```

Comment:
This is not the correct way to use async await.

```suggestion
    const response = await login(user_credentials);
```

Reply:
Also wrap it around a `try/catch`